tests/test_freqtradebot.py

```diff
@@ -1252,6 +1252,7 @@ def test_create_stoploss_order_insufficient_funds(mocker, default_conf, caplog,
 @pytest.mark.usefixtures("init_persistence")
 def test_handle_stoploss_on_exchange_trailing(mocker, default_conf, fee,
                                               limit_buy_order, limit_sell_order) -> None:
+    # TODO-lev: test for short
```

Comment:
I left these as todo's because I have another PR that I'm going to make for freqtradebot where I make a lot more changes to the stoploss, but since I had to edit `create_stoploss_order` and `handle_trailing_stoploss_on_exchange` in freqtradebot  to give the side argument, I wanted to make sure this wasn't forgotten